pkg/kube/exec_test.go

```diff
@@ -121,7 +123,7 @@ func (s *ExecSuite) TestKopiaCommand(c *C) {
 			Containers: []v1.Container{
 				v1.Container{
 					Name:  "kanister-sidecar",
-					Image: "ghcr.io/kanisterio/kanister-tools:0.37.0",
+					Image: consts.LatestKanisterToolsImage,
```

Comment:
```suggestion
					Image: "ghcr.io/kanisterio/kanister-tools:0.37.0",
```
Revert to see if test passes